tests/jballerina-unit-test/src/test/resources/testng.xml

```diff
@@ -135,6 +135,24 @@
                 </methods>
             </class>
             <class name="org.ballerinalang.test.expressions.binaryoperations.EqualAndNotEqualOperationsTest"/>
+            <class name="org.ballerinalang.test.types.decimaltype.BDecimalNaNInfinityTest">
+                <!--We can remove excludes once 'isNaN' and 'isFinite' are supported-->
```

Comment:
Are these native functions or instructions at BVM?

Reply:
IS_NAN, IS_INFINITE and IS_FINITE  should be already desugar to binary expressions. Hence should be work normally. may be some operators are missing?